### PR TITLE
Added watch-honey-badger option

### PR DESCRIPTION
**Pull request by TDVDesign**

### CODE REVIEW

**Comment by TDVDesign** [comment]
Please review this, @casey-collab

File: english/watch-honey-badger/watch-honey-badger.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+You [watch](https://www.youtube.com/watch?v=4r7wHMg5Yjg) a video about a honey badger.
```
Comment (casey-collab) [inline english/watch-honey-badger/watch-honey-badger.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by TDVDesign** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**